modules/ROOT/pages/multi-factor-authentication.adoc

```diff
@@ -0,0 +1,136 @@
+. Click *Save*. +
+All nonexempt users in your organization are now required to configure a verification method when they log in to Anypoint Platform.
+
+=== Exempt Users from Multi-factor Authentication
+
+Service users or accounts that make programmatic calls to Anypoint Platform should be exempt from multi-factor authentication because they cannot provide verification aside from a username and password.
+
+It is best to use the xref:connected-apps-overview.adoc[Connected Apps] feature to authorize apps to make programmatic calls to Anypoint Platform instead of using individual user accounts. Using the Connected Apps feature provides more security and control than using user accounts.
+
+To exempt individual accounts from multi-factor authentication:
+
+. Log in to Anypoint Platform using an account that has the Organization Administrator permission.
+. Navigate to *Access Management*.
+. In the navigation menu, click *Multi-factor Auth*.
+. In the *Exempt Accounts* section, add the accounts that you want to exempt. +
+Only users who do not use single sign-on (SSO) appear in this list.
+. Click *Save*. +
+The exempt user accounts appear in the *Exempt Accounts* section.
+
+You can later remove user accounts from the *Exempt Accounts* list by clicking *X* next to their account names.
+
+== Enable Multi-factor Authentication on Your User Account
+
+You can enable multi-factor authentication on your user account even if your organization does not require it. You can configure more than one verification method for your account:
+
+. Log in to Anypoint Platform.
+. Navigate to your user profile.
+. Click *Configure multi-factor authentication (MFA)*. +
+The *Manage Your Verification Methods* interface appears.
+. Next to the verification method you want to configure, click *Add*.
+. Follow the instructions for your preferred verification method.
+
+== Modify or Remove Multi-factor Authentication on Individual User Accounts
+
+If you have previously configured multi-factor authentication, you can modify which verification methods you use for your account. If your organization requires multi-factor authentication, you must have at least one verification method configured.
+
+If your organization has disabled its multi-factor authentication requirement, you can also remove all verification methods. However, for enhanced security, it is best to use at least one verification method.
+
+To modify verification methods on an individual user account:
+
+. Log in to Anypoint Platform.
+. Navigate to your user profile.
+. Click *Configure multi-factor authentication (MFA)*. +
+The *Manage Your Verification Methods* interface appears.
+. Add an additional verification method.
+.. Next to the verification method you want to configure, click *Add*.
+.. Follow the instructions to configure your preferred verification method.
+.. Click *Done*. +
+You can now use the configured verification method to verify your identity when you log in to Anypoint Platform.
```

Comment:
This added statement kind of bothers me because it sounds like part of the task is done, and we really don't want to imply done until the very end, right? The container step states the objective, so why confirm success here?